crates/teloxide-core/src/net.rs

```diff
@@ -35,6 +36,7 @@ pub const TELEGRAM_API_URL: &str = "https://api.telegram.org";
 ///
 /// If `TELOXIDE_PROXY` exists, but isn't correct url.
 #[must_use]
+#[cfg(not(target_family="wasm"))]
```

Comment:
Why?

Reply:
We cannot set a proxy in WASM environments